components/org.wso2.carbon.identity.api.server.action.management/org.wso2.carbon.identity.api.server.action.management.v1/src/main/resources/Actions.yaml

```diff
@@ -0,0 +1,596 @@
+openapi: 3.0.0
+info:
+  description: >
+    This document specifies an **Action Management RESTful API** for **WSO2
+    Identity Server**.
+  version: 1.0.0
+  title: WSO2 Identity Server - Action Management Rest API
+  termsOfService: 'http://swagger.io/terms/'
+  contact:
+    name: WSO2
+    url: 'http://wso2.com/products/identity-server/'
+    email: architecture@wso2.org
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+
+security:
+  - OAuth2: []
+  - BasicAuth: []
+paths:
+  /actions/types:
+    get:
+      tags:
+        - Actions
+      operationId: getActions
+      summary: |
+        Detailed summary of the Action Types
+      description: |
+        This API provides the capability to retrieve the detailed summary of the action types.<br>
+          <b>Permission required:</b> <br>
+              * /permission/admin/manage/identity/actionmgt/view <br>
```

Comment:
Why do we still introduce permission strings according to the legacy permission tree which was no longer used since IS-7.0.0?

If we are not registering these new permissions into permission tree no one can use even with legacy runtime